h2o-py/h2o/model/multinomial.py

```diff
@@ -160,7 +160,7 @@ def plot(self, timestep="AUTO", metric="AUTO", **kwargs):
         ...           y=response_col,
         ...           training_frame=train,
         ...           validation_frame=valid)
-        >>> gbm.plot(metric=AUTO, timestep=AUTO)
+        >>> gbm.plot(metric="AUTO", "timestep=AUTO")
```

Comment:
quotes around "AUTO" only.
Change this to gbm.plot(metric="AUTO", timestep="AUTO")